test/wokwi/run-simulator.sh

```
@@ -0,0 +1,31 @@
+echo "Timeout: ${WOKWI_TIMEOUT} seconds"
+echo "Web server will be available at http://localhost:8080"
+echo "Serial output will be displayed below"
+echo "=================================="
+echo ""
+
+# Run wokwi-cli with timeout (in milliseconds) and scenario flag for better output
+# The simulator will forward port 80 to localhost:9080
+# Note: wokwi-cli runs in foreground, so this needs to be backgrounded or run in a separate process
+# When run from the directory containing diagram.json and wokwi.toml, wokwi-cli will find them automatically
+# Serial output goes to stdout, diagnostic messages go to stderr
+wokwi-cli --timeout ${WOKWI_TIMEOUT}000 .
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)